build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/webpackTask.ts

```diff
@@ -32,16 +37,6 @@ export class WebpackTask extends LeafWithDoneFileTask {
 				);
 			}
 
-			const configFiles = await globFn(
-				toPosixPath(this.node.pkg.directory) + "/webpack.*.js",
-			);
-			configFiles.push(this.configFileFullPath);
-			for (const configFile of configFiles) {
-				content.config[configFile] = await this.node.buildContext.fileHashCache.getFileHash(
-					configFile,
-				);
-			}
-
```

Comment:
And this means that changes to webpack configs that aren't what gets detected as default, won't cause webpack tasks to re-run during build?

Reply:
Previously, we assume all the webpack.*.js are the only config files, which is not true.

This changes so that we would actually load the config and save the value.   If the config file change, but didn't change the values, then it won't rebuild.